components/simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,28 @@
+@import url("/docs/main.css");
+
+.grid_container {
+    display: grid;
+    background-color: var(--light-grey-color);
+    width: fit-content;
+    height: fit-content;
+}
+
+.grid_item {
+    padding: 0.5em;
+} 
+
+.grid_img {
+    height: 100%;
+}
+
+.grid_two_columns {
+    grid-template-columns: 1fr 1fr;
+    gap: 10px;
+    padding: 1rem;
+}
+
+.grid_three_columns {
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
```

Comment:
As gap:10px is used in both grid_two_columns and grid_three_columns classes, you can combine it in the grid_container class.

Reply:
Actually, I've done that intentionally to have an ease to just change gap respective to a particular grid container (based on (1*2) , (1*3) or any (x*y) ) . Otherwise, the gap will come along default with main class i.e. ".grid".
Btw, thanks for suggesting :)